### ISSUE TITLE
Feature request: view of ignored dependencies / versions

### ISSUE THREAD

**Issue by richardowen** [dependabot/feedback#72]
The ability to ignore dependencies or certain versions of dependencies is great if you can't upgrade to a certain version right now. However they then get lost.

It would be great if there was a way to see (perhaps in the Dependabot dashboard) the currently-ignored dependencies so they can be reviewed and ignore rules removed as required.

**Comment by greysteil** [comment]
Totally up for this. That dashboard is going to get an overhaul that allows you to see details of each dependency soon, and at that point I'll definitely implement this.

Thanks for the feedback!

**Comment by aldendaniels** [comment]
+1, greenkeeper lets you specifiy the ignore rules in `package.json`:
```json
"greenkeeper": {
    "ignore": [
      "bluebird",
    ]
}
```

**Comment by cihati** [comment]
@greysteil any updates on this?

**Comment by greysteil** [comment]
Not yet - we ended up prioritising config files over the update config show view, but this is next on our list.

**Comment by feelepxyz** [comment]
Yes keen to start working on this soon!

For context we should also show which dependencies can't be updated due to version constraints, e.g. peer dependencies in npm.

Raised here: https://github.com/dependabot/feedback/issues/339

**Comment by JWGmeligMeyling** [comment]
I'd like to see this too. Recently had to ignore a couple of dependencies that had to be updated together, but now I don't seem to be able to enable it again 🤷‍♂

**Comment by dannyshaw** [comment]
Is there no way at all of viewing what's been ignored? 

Can you enable a flat file download or something as an MVP? No need for pretty UI!

**Comment by greysteil** [comment]
Not at the moment. We'll integrate this into the GitHub Dependency Graph screen as we do the full integration with GitHub, though :-)

**Comment by mjoyce91** [comment]
Is this still being considered?

**Comment by feelepxyz** [comment]
@mjoyce91 yep but will take a while before we get to it. We're planning to expose this in the GitHub dependency graph view under repository insights.

**Comment by ChromeQ** [comment]
Any updates on this @feelepxyz ? You say it is being considered but this issue is almost 2 years old, is it likely to ever be resolved?

**Comment by feelepxyz** [comment]
@ChromeQ we've just launched a beta of Dependabot that's natively integrated into GitHub: https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/

If you migrated to the new config file from the [dependabot dashboard](https://app.dependabot.com/) we'll add the current ignored dependencies to the config file giving you more visibility.

We're working on exposing which versions are being ignored in the logs. Providing a full UI view of this will be a big lift for us as we currently don't persist the project's dependencies and current/latest version.

**Comment by ChromeQ** [comment]
Thanks, sounds good. 
But I am trying to enable it and the config file has been committed, but I do not have the Security & analysis tab under my settings. Is this enabled for private repos? I can't see it in one of my public ones either

**Comment by ChromeQ** [comment]
I found it, it is not under the Settings tab, but take a look under the Security tab and turn on Dependabot alerts.
